eva-accession-core/src/test/java/uk/ac/ebi/eva/accession/core/SubmittedVariantAccessioningServiceTest.java

```diff
@@ -218,6 +218,41 @@ public void getOrCreateAccessionsInBothRepositories() throws AccessionCouldNotBe
         assertEquals(3, submittedVariants.size());
     }
 
+    @UsingDataSet(locations = {"/test-data/dbsnpSubmittedVariantEntity.json"})
+    @Test
+    public void getOrCreateDbsnpIdenticalVariants() throws AccessionCouldNotBeGeneratedException {
+        SubmittedVariant variantPresentInDbsnp = new SubmittedVariant("GCA_000009999.3", 9999, PROJECT_DBSNP, "21",
+                                                                      20849999, "", "GG", null);
+
+        List<AccessionWrapper<ISubmittedVariant, String, Long>> generatedAccessions = service.getOrCreate(
+                Collections.singletonList(variantPresentInDbsnp));
+        assertEquals(1, generatedAccessions.size());
```

Comment:
Would be better to test the accession itself (ACCESSION_DBSNP_1) rather than the number of generated accessions.